tests/unit_tests/thumbnails/test_digest.py

```diff
@@ -131,45 +152,103 @@ def CUSTOM_CHART_FUNC(
             [ExecutorType.CURRENT_USER],
             True,
             False,
+            [],
             "5380dcbe94621a0759b09554404f3d02",
         ),
         (
             None,
             [ExecutorType.CURRENT_USER],
             True,
+            False,
+            [
+                {
+                    "is_rls_supported": True,
+                    "get_sqla_row_level_filters": MagicMock(return_value=["filter1"]),
+                }
+            ],
+            "f4f262a649225d62717cdb11a9f2b8ee",
+        ),
+        (
+            None,
+            [ExecutorType.CURRENT_USER],
+            True,
+            False,
+            [
+                {
+                    "is_rls_supported": True,
+                    "get_sqla_row_level_filters": MagicMock(
+                        return_value=["filter1", "filter2"]
+                    ),
+                },
+                {
+                    "is_rls_supported": True,
+                    "get_sqla_row_level_filters": MagicMock(
+                        return_value=["filter3", "filter4"]
+                    ),
+                },
+            ],
+            "4903ad13cfe28a5fc41478147d87211a",
+        ),
+        (
+            None,
+            [ExecutorType.CURRENT_USER],
             True,
-            "1.current_user.1",
+            False,
+            [
+                {
+                    "is_rls_supported": False,
+                    "get_sqla_row_level_filters": MagicMock(return_value=[]),
+                },
+                {
+                    "is_rls_supported": True,
+                    "get_sqla_row_level_filters": MagicMock(
+                        return_value=["filter1", "filter2"]
+                    ),
+                },
+            ],
+            "774b0c1ed98142ee8f60aae51f128000",
         ),
         (
             None,
             [ExecutorType.CURRENT_USER],
             False,
             False,
+            [],
             ExecutorNotFoundError(),
         ),
     ],
 )
-def test_dashboard_digest(
+def test_dashboard_digest_with_rls(
```

Comment:
Not sure if we need to append `with_rls` to the test name, as we don't have a case where we want to ignore RLS in digest creation